plugins/dfu/dfu-target.c

```diff
@@ -1285,6 +1321,15 @@ dfu_target_download (DfuTarget *target, DfuImage *image,
 			return FALSE;
 	}
 
+	if (fu_device_has_custom_flag (FU_DEVICE (dfu_target_get_device (target)),
+				       "poll-in-manifest") &&
+		dfu_device_has_attribute (priv->device,
+						DFU_DEVICE_ATTRIBUTE_MANIFEST_TOL)) {
+		ret = dfu_target_poll_in_manifest (target, error);
+		if (!ret)
```

Comment:
I think lose the `ret` temp var and just put on one line

Reply:
Yeah. I will change it.